include/swift/AST/Types.h

```diff
@@ -338,14 +338,14 @@ class alignas(1 << TypeAlignInBits) TypeBase {
 
   SWIFT_INLINE_BITFIELD_FULL(TypeVariableType, TypeBase, 64-NumTypeBaseBits,
```

Comment:
You need to update this line too, to say how many bits are actually used. (That detects overflow.)

Reply:
The previous version of the line was wrong, too. It should have just been `32+4+28`, with no subtraction. Now it should be `NumTypeBaseBits+20+4+20`.

Reply:
Ah alright, I'm going to adjust that, I though that this means that in total it uses 64 bits - what ever 20 bit for TypeBase and 44 for the rest, which kind of fits into `64 - NumTypeBaseBits`

Reply:
But it's definitely better to be explicit about it.

Reply:
Actually that should be just `20 + 4 + 20` since `C` in that template is extra bits :)

Reply:
Oops, you're right. I didn't realize it was just the local bits.

Reply:
Alright, modified it to be explicit.